### ISSUE TITLE
Poor correlation with coastlines

### ISSUE THREAD

**Issue by jolyonralph** [aourednik/historical-basemaps#19]
There seems to be a poor correlation with coastlines in many cases - for example going to 1815-1920 and looking at the United Kingdom of Great Britain and Ireland, a lot of the coastal areas (and many of the surrounding islands including the Orkneys, Shetlands, Isle of Wight and Anglesey) are excluded.

I appreciate that you want to keep the polygon size small, but this would be better done by drawing the coastal borders around the island(s) rather than inside. It won't be an accurate capture of territorial waters of the time but what it will do is capture correctly all coordinates within the territory on land, which the current data fails to do.

I'm looking at this as a way of being able to map existing data points on land and see which territories they fell into in the historical past - but these polygons currently don't do this effectively because they are clipping out significant parts of coastal regions. 

Is there any intent to fix this, or will I have to fork the data and work on this myself?

**Comment by jolyonralph** [comment]
Or, if it's helpful, I'm happy to contribute such edits back to this project directly - what is the recommended geojson editor for working on these files?

**Comment by aourednik** [comment]
@jolyonralph : I totally agree. The islands, especially, often played - and still do - important strategic roles in the demarcotory process of territorial waters, something that would also need to be added. The original dataset is indeed poorely aligned with coastal lines; I've already amended it in some cases, like the Carribean or Danemark, for some years.
Your help would be much appreciated. I recommend editing with the open-source tool [QGIS](https://www.qgis.org). However: doing this by hand represents a tremendous amount of work. Best woul be to find an _automatized_ way to snap the current data to better-defined continetal polygons. QGIS provides a [geometry snapping python class](https://qgis.org/api/3.4/classQgsInternalGeometrySnapper.html) that could allow to write a script for doing so; I did not have time to give it a closer look yet, though.
